app/user/user_model.py

```diff
@@ -0,0 +1,31 @@
+from app import db
+from werkzeug.security import generate_password_hash, check_password_hash
+from enum import Enum
+
+class UserRole(Enum):
+    STANDARD = 'standard'
+    COMMITTEE = 'committee'
+    ADMIN = 'admin'
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    firstName = db.Column(db.String(64), index=True, unique=False)
+    lastName = db.Column(db.String(64), index=True, unique=False)
+    email = db.Column(db.String(120), index=True, unique=True)
+    password_hash = db.Column(db.String(128))
+    role = db.Column(db.String(64), index=False, unique=False)
+
+    """role = db.Column(db.Enum(UserRole), default=UserRole.STANDARD) 
+    would line 18 be better? 'Since you've already defined an Enum for 
+    user roles (UserRole), it would be better to use that enum directly 
+    in your model'"""
```

Comment:
We can set the default to standard but that essentially hides the implementation. Normally this is ok to prevent no value being set but since we want to be explicit when creating the user in `UserService` we won't want to use this approach. Lets remove the comment for this PR